services/user.service.js

```diff
@@ -0,0 +1,150 @@
+const filename = process.cwd() + "/data/movies.json";
+const fs = require("fs");
+let movies = require(filename);
+const helper = require("../helpers/helper.js");
+
+/**
+ * This function returns all the movies
+ *
+ * @returns {Array} Returns all the movies
+ */
+function getMovies() {
+  return new Promise((resolve, reject) => {
+    if (movies.length === 0) {
+      reject({
```

Comment:
Rejection usually means an error of some kind. Here though lack of movies in the store is not an error, so just resolve with empty array and leave the client to decide how to handle it.